M2/Macaulay2/d/evaluate.d

```diff
@@ -1305,11 +1305,11 @@ steppingFurther(c:Code):bool := steppingFlag && (
      if p == dummyPosition || p.loadDepth < errorDepth then return true;
      if stepCount >= 0 then (
 	  if lastCodePosition.filename != p.filename
-	  || lastCodePosition.line != p.line
+	  || lastCodePosition.lineF != p.lineF
 	  then (
      	       stepCount = stepCount - 1;
      	       lastCodePosition.filename = p.filename;
-	       lastCodePosition.line = p.line;
+	       lastCodePosition.lineF = p.lineF;
```

Comment:
This is another place where I'm not sure if the correct location should be left-point or focused-point. Technically all three were the same before, so which one is important here?

Reply:
I'd stick to focus for now. one may want to rewrite some of this to take advantage of the 3 positions, but I don't think there's any rush to do so.